### PR TITLE
New-HTMLTab - Add Text-Transform

### PR DESCRIPTION
**Pull request by xlrod**
Added the `$TextTransform` parameter to the New-HTMLTab function to allow the text to be uppercase, lowercase or capitalized.

`New-HTMLTab -Name "Main" -IconRegular map -TextTransform capitalize {}`

tab

capitalize

### CODE REVIEW

File: Public/New-HTMLTab.ps1 [hunk 1]
```diff
@@ -68,7 +68,8 @@ function New-HTMLTab {
         [parameter(ParameterSetName = "FontAwesomeSolid")][int] $IconSize,
         [parameter(ParameterSetName = "FontAwesomeBrands")]
         [parameter(ParameterSetName = "FontAwesomeRegular")]
-        [parameter(ParameterSetName = "FontAwesomeSolid")][string] $IconColor
+        [parameter(ParameterSetName = "FontAwesomeSolid")][string] $IconColor,
+        [ValidateSet('uppercase', 'lowercase', 'capitalize')][string] $TextTransform  # New-HTMLTab - Add text-transform
```
Comment (PrzemyslawKlys) [inline Public/New-HTMLTab.ps1]: Why not define = 'uppercase' as default here so you can skip if ($TextTransform) ? Also the default now is "none" right? With your change you want uppercase if nothing is selected?

Comment (xlrod) [reply Public/New-HTMLTab.ps1]: Good catch! Yes, I wanted to keep the 'uppercase' as the default and set to other values if selected.

File: Public/New-HTMLTab.ps1 [hunk 2]
```diff
@@ -90,6 +91,14 @@ function New-HTMLTab {
     if ($TextColor) {
         $StyleText.'color' = ConvertFrom-Color -Color $TextColor
     }
+    # New-HTMLTab - Add text-transform
+    if($TextTransform){
```
Comment (xlrod) [inline Public/New-HTMLTab.ps1]: Removed the if/else statement and left `$StyleText.'text-transform' = "$TextTransform"`

File: Public/New-HTMLTab.ps1 [hunk 3]
```diff
@@ -90,6 +91,14 @@ function New-HTMLTab {
...
+        $StyleText.'text-transform' = "$TextTransform"
+    }
+    else{
+        $StyleText.'text-transform' = "uppercase"
+    }
+    # end
```
Comment (xlrod) [inline Public/New-HTMLTab.ps1]: Removed the if/else statement and left `$StyleText.'text-transform' = "$TextTransform"`

Comment (PrzemyslawKlys) [reply Public/New-HTMLTab.ps1]: You only reviewed, but didn't submit any code changes :-)

Comment (xlrod) [reply Public/New-HTMLTab.ps1]: I'm sorry, I'm new with GitHub. Should I attach the modified code or wait for the Request Changes?

Comment (PrzemyslawKlys) [reply Public/New-HTMLTab.ps1]: you can simply modify code and push code changes. Until you actually change branch it should keep going into this PR.

**Comment by PrzemyslawKlys** [comment]
Tnx ;)

**Pull request merged**